coordinator/tests/services/translation/dispention/release.spec.ts

```diff
@@ -0,0 +1,34 @@
+import * as fhir from "../../../../src/models/fhir/fhir-resources"
+import {translateReleaseRequest} from "../../../../src/services/translation/dispention/release"
+
+describe("translateReleaseRequest", () => {
+  const parameters = new fhir.Parameters([{
+    "name": "owner",
+    "valueIdentifier": {
+      "system": "https://fhir.nhs.uk/Id/ods-organization-code",
+      "value": "VNE51"
+    }
+  }])
+  const translatedRelease = translateReleaseRequest(parameters)
+
+  test("translated release contains AgentPerson", () => {
+    const author = translatedRelease.NominatedPrescriptionReleaseRequest.author
+    expect(author).not.toBeUndefined()
+    const agentPerson = author.AgentPerson
+    expect(agentPerson).not.toBeUndefined()
+  })
+
+  test("translated release contains agentPersonPerson and representedOrganization", () => {
+    const agentPerson = translatedRelease.NominatedPrescriptionReleaseRequest.author.AgentPerson
+    const agentPersonPerson = agentPerson.agentPerson
+    const representedOrganization = agentPerson.representedOrganization
+    expect(agentPersonPerson).not.toBeUndefined()
+    expect(representedOrganization).not.toBeUndefined()
+  })
+
+  test("", () => {
```

Comment:
translates organizationId correctly

Reply:
potentially worth having a test around short form id too? or is that not scoped atm